Pipfile

```
@@ -8,9 +8,11 @@ python_version = "3.9"
 
 [packages]
 dominate = "~=2.6.0"
+click = "8.1.3"
```

we have all the other dependencies like this:
```suggestion
click = "~=8.1.3"
```

---

Please run `pipenv uninstall click` and `pipenv install click~=8.1.3` to also update the `Pipfile.lock`

---

ack